docs/publications.rst

```diff
@@ -0,0 +1,92 @@
+.. _Publications:
+
+************
+Publications
+************
+
+2022
+====
+
+Eytan J. Adler, Benjamin J. Brelje, and Joaquim R.R.A. Martins,
+*Thermal Management System Optimization for a Parallel Hybrid Aircraft Considering Mission Fuel Burn*,
+Aerospace, 9(5), 2022
+DOI: `10.3390/aerospace9050243 <https://doi.org/10.3390/aerospace9050243>`_.
+PDF is available `here <https://mdolab.engin.umich.edu/bibliography/Adler2022b>`__.
+
+.. code-block:: bibtex
+
+	@article{Adler2022b,
+		article-number = {243},
+		author = {Adler, Eytan J. and Brelje, Benjamin J. and Martins, Joaquim R. R. A.},
+		doi = {10.3390/aerospace9050243},
+		issn = {2226-4310},
+		journal = {Aerospace},
+		keywords = {OpenMDAO, OpenConcept},
+		month = apr,
+		number = {5},
+		title = {Thermal Management System Optimization for a Parallel Hybrid Aircraft Considering Mission Fuel Burn},
+		volume = {9},
+		year = {2022}
+	}
+
+Eytan J. Adler and Joaquim R.R.A. Martins,
+*Aerostructural wing design optimization considering full mission analysis*,
+2022 AIAA SciTech Forum, San Diego, CA, January 2022,
+DOI: `10.2514/6.2022-0382 <https://doi.org/10.2514/6.2022-0382>`_.
+PDF is available `here <https://mdolab.engin.umich.edu/bibliography/Adler2022a>`__.
+
+.. code-block:: bibtex
+
+	@inproceedings{Adler2022a,
+		author = {Eytan J. Adler and Joaquim R. R. A. Martins},
+		title = {Aerostructural wing design optimization considering full mission analysis},
+		booktitle = {AIAA SciTech Forum},
+		doi = {10.2514/6.2022-0382},
+		month = {January},
+		year = {2022}
+	}
+
+2019
+====
+
+Benjamin J. Brelje, John P. Jasa, Joaquim R.R.A. Martins, and Justin S. Gray,
+*Development of a Conceptual-Level Thermal Management System Design Capability in OpenConcept*,
+NATO Research Symposium on Hybrid/Electric Aero-Propulsion Systems for Military Applications (AVT-RSY-323), 2019,
+DOI: `10.14339/STO-MP-AVT-323 <https://doi.org/10.14339/STO-MP-AVT-323>`_.
```

Comment:
When I try to access this link it says "DOI Not Found"

Reply:
The PDF download works as expected.

Reply:
NATO's gotta get it together. I'll leave just the PDF